MIDIchlorians/MIDIchlorians/Info.plist

```diff
@@ -14,28 +14,42 @@
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
 	<key>CFBundleURLTypes</key>
 	<array>
 		<dict>
+			<key>CFBundleURLName</key>
+			<string></string>
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>db-8a5t7h2dpsfjunc</string>
 			</array>
-			<key>CFBundleURLName</key>
-			<string></string>
 		</dict>
 	</array>
-	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
-	<key>LSRequiresIPhoneOS</key>
-	<true/>
+	<key>Fabric</key>
+	<dict>
+		<key>APIKey</key>
+		<string>56683ad8cc8f1cf4ed01f5000ee9e316e315fc62</string>
```

Comment:
maybe we can try this? https://twittercommunity.com/t/how-to-prevent-publication-of-fabric-api-key-in-open-source-xcode-projects/69250